index.js

```diff
@@ -0,0 +1,58 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let div = document.createElement("div")
+  div.id = "cards"
+
+  const fetchData = (url, callback) => {
+    fetch(url)
+      .then(res => {
+        if (!res.ok) {
+          throw Error(res.statusText)
+        }
+        return res.json()
+      })
+      .then(res => {
+        callback(res)
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }
+
+  const deckId = data => {
```

Comment:
I recommend to have the deckId outside this scope, and have it on its own variable.